build.gradle

```diff
@@ -87,6 +87,7 @@ project(':dyno-jedis') {
         compile project(':dyno-core')
         compile project(':dyno-contrib')
         compile 'redis.clients:jedis:2.9.0'
+        testCompile 'com.netflix.spinnaker.embedded-redis:embedded-redis:0.8.0'
```

Comment:
If embedded-redis doesn't work on Windows, use conditional ignore (https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html) to ignore such tests

Reply:
Also I am curious to know why embedded-redis doesn't work on windows

Reply:
It looks like a change introduced on the Spinnaker [fork](https://github.com/spinnaker/embedded-redis) of [kstyrc/embedded-redis](https://github.com/kstyrc/embedded-redis) (which smelled abanoneded to me).  I saw it noted on the [README](https://github.com/spinnaker/embedded-redis#redis-version) of the Spinnaker fork.

The reason is the library actually embeds OS-specific executables that are executed as separate processes.

I will follow your recommendation to conditionally ignore this test and make sure things are green on Windows.